Tests/DangerTests/SwiftLint/DangerSwiftLintTests.swift

```diff
@@ -389,15 +389,30 @@ final class DangerSwiftLintTests: XCTestCase {
                            readFile: mockedViolationJSON)
         XCTAssertNotNil(markdownMessage)
 
-        let lines = markdownMessage!.split(separator: "\n")
-        XCTAssertEqual(lines[3],
+        let lines = markdownMessage?.split(separator: "\n")
+        XCTAssertEqual(lines?[3],
                        "Error | SomeFile.swift:8 | " +
                            "Opening braces should be preceded by a single space and on the same line as the declaration." +
                            " (`opening_brace`) |")
-        XCTAssertEqual(lines[4],
+        XCTAssertEqual(lines?[4],
                        "Error | AnotherFile.swift:10 | Line should be 120 characters or less: currently 211 characters (`line_length`) |")
     }
 
+    func testMarkdownReportingWithoutFilePath() {
+        _ = SwiftLint.lint(danger: danger,
+                           shellExecutor: executor,
+                           swiftlintPath: "swiftlint",
+                           strict: true,
+                           currentPathProvider: fakePathProvider,
+                           markdownAction: writeMarkdown,
+                           readFile: mockedViolationJSONWitNoFile)
+        XCTAssertNotNil(markdownMessage)
+
+        let lines = markdownMessage?.split(separator: "\n")
+        XCTAssertEqual(lines?[3],
+                       "Error |  | Opening braces should be preceded by a single space and on the same line as the declaration. (`opening_brace`) |")
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/SwiftLint/DangerSwiftLintTests.swift;
  Line: 413;
-->

- :warning: Line should be 140 characters or less: currently 149 characters (`line_length`)